README.md

```diff
@@ -77,6 +77,12 @@ For older versions (python 2.7, <3.4), you can continue using gpustat v0.x.
 [gh-issue-66]: https://github.com/wookayin/gpustat/issues/66
 
 
+Tests
+-----
+
+To run the test suite, execute `PYTEST_ADDOPTS="-s" python setup.py test`
```

Comment:
I'm not sure what the differences are, but I know of at least one:
`pytest .` doesn't work if I don't have pytest installed locally (naturally). `python setup.py test` does work because it installs the testing dependencies as eggs, so it's self-contained. This is the command that's being run on travis. I suspect it proceeds to run exactly `pytest .` after that via the egg, as you suggest.